resources/user-agents/browsers/mqq-browser/mqq-browser-2-7-on.json

```diff
@@ -27,29 +27,30 @@
       },
       "children": [
         {
-          "match": "MQQBrowser/#MAJORVER#.#MINORVER#*Mozilla/5.0 (#PLATFORM#GT-N7100 Build/*) applewebkit* (*khtml*like*gecko*) Version/4.0 *Safari/*",
```

Comment:
I ran into some agents that were detecting as "Nokia" (more on that later) there were QQ Browser, so I just condensed the devices while I was in here.